test/e2e/services.go

```diff
@@ -53,10 +56,40 @@ func startServicesForMetrics(t *testing.T, e e2e.Environment) (
 func startServicesForRules(t *testing.T, e e2e.Environment) (metricsRulesEndpoint string) {
 	// Create S3 replacement for rules backend
 	bucket := "obs_rules_test"
-	m := e2edb.NewMinio(e, "rules-minio", bucket)
-	testutil.Ok(t, e2e.StartAndWaitReady(m))
+	userID := strconv.Itoa(os.Getuid())
+	ports := map[string]int{e2edb.AccessPortName: 8090}
+	envVars := []string{
+		"MINIO_ROOT_USER=" + e2edb.MinioAccessKey,
+		"MINIO_ROOT_PASSWORD=" + e2edb.MinioSecretKey,
+		"MINIO_BROWSER=" + "off",
+		"ENABLE_HTTPS=" + "0",
+		// https://docs.min.io/docs/minio-kms-quickstart-guide.html
+		"MINIO_KMS_KES_ENDPOINT=" + "https://play.min.io:7373",
+		"MINIO_KMS_KES_KEY_FILE=" + "root.key",
+		"MINIO_KMS_KES_CERT_FILE=" + "root.cert",
+		"MINIO_KMS_KES_KEY_NAME=" + "my-minio-key",
+	}
+	f := e2e.NewInstrumentedRunnable(e, "rules-minio", ports, e2edb.AccessPortName)
+	runnable := f.Init(
+		e2e.StartOptions{
+			Image: "minio/minio:RELEASE.2022-03-03T21-21-16Z",
```

Comment:
Why not latest release?

Reply:
@matej-g did you mean `:latest` or that I should have picked up the latest release. If it is the latter, I think you are correct and have done so in https://github.com/efficientgo/e2e/pull/26

I initially tested with the release I felt was going to fix the issue.

Reply:
I meant whatever is the latest tagged release, not `latest`, if I seen correctly there is a newer version. Thanks @PhilipGough :+1: